.github/workflows/self-scheduled-amd-mi300-caller.yml

```diff
@@ -0,0 +1,29 @@
+name: Self-hosted runner (AMD mi300 scheduled CI caller)
+
+on:
+  workflow_run:
+    workflows: ["Self-hosted runner (AMD scheduled CI caller)"]
+    branches: ["main"]
+    types: [completed]
+  push:
+    branches:
+      - run_amd_scheduled_ci_caller*
+      - mi300-ci
+
+jobs:
+  build-docker-containers:
+    # TODO: remove this
```

Comment:
@fxmarty Before I merge, could you remove this and any other places that should be removed if any.

Thanks!

Reply:
gently ping @fxmarty  🙏